CONTRIBUTING.md

```diff
@@ -153,6 +153,17 @@ At this point the pre-commit checks will run against any files that
 you attempt to commit.  If you want to run the checks against the
 entire repo, just execute `pre-commit run --all-files`.
 
+## Quality Assurance/Code reviews ##
```

Comment:
[We only capitalize the first word of secondary headings](https://github.com/cisagov/development-guide/blob/develop/style-guide.md#headers-and-headings):
```suggestion
## Quality assurance/code reviews ##
```